e2e-test/cypress/integration/documents_spec.js

```diff
@@ -54,13 +58,19 @@ describe("Attaching a document to a workflowitem.", function() {
     const fileDisplayName = "Validation_Test";
     uploadDocument(fileDisplayName, fileName);
     // submit and close the dialog:
-    cy.get("[data-test=submit]").click();
+    cy.get("[data-test=submit]")
+      .should("be.visible")
+      .click();
 
```

Comment:
wait for subproject.viewdetails before opening the info dialog. If it is too fast the document isn't shown in the documents tab.

Reply:
route and wait is now set